src/components/StorageOverview/OCSHealth/HealthBody.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+const healtyIcon = (
+  <div className="kubevirt-ocs-health__icon--ok">
+    <Icon type="fa" size="2x" name="check-circle" />
+  </div>
+);
+
+const errorIcon = (
+  <div className="kubevirt-ocs-health__icon--error">
+    <Icon type="fa" size="2x" name="exclamation-circle" />
+  </div>
+);
+
+const warningIcon = (
+  <div className="kubevirt-ocs-health__icon--warning">
+    <Icon type="fa" size="2x" name="exclamation-triangle" />
+  </div>
+);
+
+const OCSHealthMap = {
```

Comment:
maybe it would make sense to make a status component out of this

Reply:
+1 lets do this when we refactor Health Card